pyomo/solvers/plugins/solvers/SCIPAMPL.py

```diff
@@ -304,6 +411,10 @@ def _postsolve(self):
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.unbounded
+                    
+        # # WARNING
+        # infeasibleOrUnbounded='infeasibleOrUnbounded'   # Problem is either infeasible or unbounded
```

Comment:
I appreciate these comments being inserted, but can they please be cleaned up a bit? I.e., one comment per line?